lib/versioning/cargo/index.spec.ts

```diff
@@ -1,16 +1,15 @@
 import { api as semver } from '.';
 
-describe('semver.matches()', () => {
```

Comment:
better add a new nesting level, so we can have function name as second level describe block

Reply:
In this PR, I've made minimal merge into single top-level `describe`-block. Later, I would like to gradually refactor all versioning tests to more readable table form, as I did for Cargo. And I'm about to start from version schemes you mentioned here.

Reply:
I would prefer to add a new toplevel  describe block here instead, i can  ignore whitespace changes at review, so we have only a few changed lines.

Reply:
Ah, now I got what you wanted. Just was confused with `it()` that doesn't allow duplicates and thought nested `describe` would raise similar lint warnings too.